server/image_list.go

```diff
@@ -36,6 +36,8 @@ func ConvertImage(from *storage.ImageResult) *pb.Image {
 	repoTags := []string{"<none>:<none>"}
 	if len(from.RepoTags) > 0 {
 		repoTags = from.RepoTags
+	} else if from.PreviousName != "" {
+		repoTags = []string{from.PreviousName + ":<none>"}
```

Comment:
I didn't test it but could this lead to situations where we return a wrong (outdated) repoTags? (I didn't look at the call sites)

Reply:
Hm, I don't think so because `<none>` will be always treated as "tag does not exist"